spot-client/src/spot-remote/ui/components/remote-control-menu/menu-button.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const MenuButton = ({ label, onClick }) => (
+    <button
```

Comment:
I do have a nav button that provides similar stylings to the other buttons in the remote-control. Would it be tricky to try to re-use that?

Reply:
I tried, but that's just way too different. But let's see what Yana suggests about this. If she wants to go with a rounded button similar to the main nav buttons, we can replace them

Reply:
Alright, I'll give a shot at refactoring the CSS a bit to support this. My mess, my burden.